.travis.yml

```diff
@@ -22,7 +22,7 @@ addons:
 install:
   - git clone https://github.com/CGAL/cgal.git
   - cd cgal
-  - git checkout releases/CGAL-4.9
+  - git checkout releases/CGAL-4.10
```

Comment:
With a configuration matrix and an environment variable. @maxGimeno knows how to setup Travis that way. Maxime, could you propose a PR for that?